src/test/java/io/confluent/examples/streams/ApplicationResetIntegrationTest.java

```diff
@@ -134,7 +134,7 @@ public void shouldReprocess() throws Exception {
     }
 
     // reset application
-    final int exitCode = new StreamsResetter().run(
+    final int exitCode = new StreamsResetter().execute(
```

Comment:
This method was renamed what broke the build.

Reply:
FYI: https://github.com/apache/kafka/pull/13983#issuecomment-1749310791
Using the `StreamsResetter` class directly is actually not the intended use.

Reply:
We use `StreamsResetter` directly in multiple places like the benchmarks, I think. So I am fine with it. That also means that we might run into breaking changes once in a while.